public/app.scss

```diff
@@ -44,3 +47,10 @@ $euiTextColor: $euiColorDarkestShade !default;
 .state-accordion:hover {
   text-decoration: none;
 }
+
+.euiFilterGroup__popoverPanel {
```

Comment:
Should we use a custom class so that only this particualr popover is styled differently?

Reply:
well, this plugin only has this specific classed popover in 2 places, and they both needed the same fix.  There is no actual code element to take a custom class name, it is auto generated from the options within the search bar filters.

Reply:
I checked every page for this class in dev tools, only comes up with the cases addressed in the PR

Reply:
![Screen Shot 2022-11-29 at 3 24 44 PM](https://user-images.githubusercontent.com/111526961/204670621-972a849b-d42e-47db-9077-b6ffd36f6969.png)

Reply:
getDataStreams returns an array of { value: _data stream name_}

Reply:
okay, let's keep it as is then